pkg/terraform/terraform.go

```diff
@@ -0,0 +1,35 @@
+package terraform
+
+import (
+	"bufio"
+	"io/ioutil"
+
+	"github.com/deislabs/porter-terraform/pkg/kubernetes"
+	"github.com/deislabs/porter/pkg/context"
+	"github.com/pkg/errors"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+// terraform is the logic behind the terraform mixin
+type Mixin struct {
+	*context.Context
+	ClientFactory kubernetes.ClientFactory
```

Comment:
Yeah, let's get rid of this field and anything that uses it. Then rerun dep ensure and ideally anything kubernetes related should fall out of our Gopkg.lock and vendor.